ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/layout.tsx

```diff
@@ -106,7 +106,8 @@ export default function FunctionLayout({ children, params }: FunctionLayoutProps
                   btnAction={invokeAction}
                 />
                 <PauseFunctionButton functionSlug={functionSlug} disabled={isArchived} />
-                <ArchiveFunctionButton functionSlug={functionSlug} />
+                {/* <ArchiveFunctionButton functionSlug={functionSlug} /> */}
```

Comment:
Any particular reason not to delete the entire Archive stuff?

Reply:
Nope -- I forgot to fully delete it!